src/pkg/utils/io.go

```diff
@@ -404,10 +404,14 @@ func SplitFile(srcFile string, chunkSizeBytes int) (err error) {
 			if err != nil {
 				return err
 			}
+			err = chunkFile.Close()
+			if err != nil {
+				return err
+			}
 
 			// create new file
-			path = fmt.Sprintf("%s.part%03d", srcFile, len(fileNames)+1)
-			chunkFile, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY, 0644)
+			path = fmt.Sprintf("%s.part%03d", srcPath, len(fileNames)+1)
+			chunkFile, err = os.OpenFile(path, os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
That's unfortunate. I had fixed this at one point. Seemed to have lost this somehow. Thanks for fixing it!